### PR TITLE
Donor can create a foodbag

### PR DESCRIPTION
**Pull request by FlisAnn**

### CODE REVIEW

**Comment by FlisAnn** [comment]
[DONOR: Donor can create a foodbag](https://www.pivotaltracker.com/story/show/176626707)
### User Story
```
As a donor,
In order to help mums
I want to be able to create a foodbag
```
## Changes proposed in this pull request:
* Donor can add a foodbag
* Skeleton of NavBarMenu for navigation

## What I have learned working on this feature:
* How to create a foodbag

## Screenshots (Client)
Screenshot from 2021-01-29 10-02-07

File: src/components/MainPageDisplay.jsx [hunk 1]
```diff
@@ -1,13 +1,16 @@
...
+import CreateFoodBagForm from './CreateFoodBagForm'
 
 const MainPageDisplay = () => {
   return (
     <Grid celled='internally'>
       <Grid.Row>
         <Grid.Column width={3}>
-          <Image src='./img/wecare_logo.png' />
+          <SideBarMeny />
+
```
Comment (psnoya) [inline src/components/MainPageDisplay.jsx]: fORMAT YOUR CODE PLEASE :) <3

File: src/components/RegistrationForm.jsx [hunk 2]
```diff
@@ -20,7 +20,11 @@ const RegistrationForm = () => {
           dispatch({ type: 'REGISTRATION_FORM', payload: { openModal: true } })
         }
         open={openModal}
-        trigger={<Button data-cy='register-btn'>Register Button</Button>}
+        trigger={<Button data-cy='register-btn'
```
Comment (psnoya) [inline src/components/RegistrationForm.jsx]: Code formatting x2

**Review by psnoya — COMMENTED** [review]
Cool BEANS! Good job

File: src/components/MainPageDisplay.jsx (hunk 1 above)
Comment (Ehugo2000) [reply src/components/MainPageDisplay.jsx]: done!

File: src/components/RegistrationForm.jsx (hunk 2 above)
Comment (Ehugo2000) [reply src/components/RegistrationForm.jsx]: done!

**Pull request merged**